Added "stunt to chat" button

alxnsk opened the pull request:
Hi, I wanted this feature, and I feel like stunt "short description" field is forgotten, so I have made it.

The user can now click on the button in title of a stunt to send its "title", "short description" and "description" to the chat.
![stunt button](https://github.com/anvil-vtt/FateX/assets/137091674/8b443a2b-d721-4405-aaeb-18b706e40089)

I had to use an unbind() call in event listeners, because somehow specifically in stunts activateActorSheetListeners binds this click event twice. Couldn't yet trace the source and I am not sure this is even a bug, so this is a "future proof" workaround.

PatrickBauer commented:
Thank you very much, I'll check this out and have a look why the unbind was necessary! :)

alxnsk commented:
Hi, I have tinkered  with this feature a little more. And I done three things:
1) Added same button to "extra", I forgot about it.
2) Changed the class  and function names. Now they are not so specific about "stunts"
3) While I was there I explicitly set async:"false" in all .enrichHTML calls if it is not set to "true". In Foundry 10 we got warning and in Foundry 11 default behaivor of this function was reversed, and it returns [object Object] instead of text. This should fix the problem.

PatrickBauer commented:
Sorry this took so long. I merged the feature into its own branch where I'll have to replace the inline html with templates and maybe add some CSS, before I release it into the main branch. Thank you very much for this contribution! 👍
